integration/pushdown.go

```diff
@@ -0,0 +1,38 @@
+package integration
+
+import (
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/internal/util/testutil/testtb"
+)
+
+// resultPushdown stores the information about expected pushdown results for a single or multiple backends.
+// For example if both pg and SQlite backends are expected to pushdown, the `pgPushdown + sqlitePushdown` operation
+// can be used.
+type resultPushdown uint8
+
+const (
+	noPushdown     resultPushdown = 0 // 0000 0000
```

Comment:
The idiomatic way is to use `iota` with a bitshift: `1 << iota`. See also all other our "enum"s where we omit type after the first constant, etc